quinn-proto/src/connection/mod.rs

```diff
@@ -2436,6 +2525,7 @@ impl Connection {
         number: u64,
         payload: Bytes,
     ) -> Result<(), TransportError> {
+        self.reset_rtt_timer(now);
```

Comment:
The function name is misleading here... It is actually setting up the RTT timer for the first time. Do you think that belongs to a different place in the code? Or would it be enough to rename it to something like `setup_rtt_timer`?

Reply:
Since the last relevant signal considered is whether the peer has declared support for the extension in its transport parameters, how about we do this when we process the peer's 1-RTT transport parameters, in `Connection::handle_peer_params`? Probably along with renaming it for clarity.

Reply:
Will do, `handle_peer_params` definitely sounds like the right place